src/components/auth/Register.js

```diff
@@ -75,8 +77,8 @@ export const Register = (props) => {
                     <input ref={verifyPassword} type="password" name="verifyPassword" className="form-control" placeholder="Verify password" required />
                 </fieldset>
                 <fieldset>
-                    <label htmlFor="verifyPassword"> Verify Password </label>
-                    <textarea ref={bio} name="bio" className="form-control" placeholder="Let other gamers know a little bit about you..." />
+                    <label htmlFor="verifyPassword"> Choose a Display Name </label>
```

Comment:
Could you change the label 'htmlFor' to indicate that it's labeling the display name field?